runtime/dancebox/src/lib.rs

```diff
@@ -2158,7 +2158,7 @@ impl_runtime_apis! {
                     // We only care for native asset until we support others
                     // TODO: refactor this case once other assets are supported
                     vec![Asset{
-                        id: AssetId(Location::here()),
```

Comment:
Why was this changed?

Reply:
For some reason it seems xcm benchmarks are now more properly tested and our configuration was wrong. We were putting in holding a token referenced by `Here` but we dont have any currency matching such multilocation. And then we were marking as `fee_asset` the native token. Again for some reason (I guess the benchmark upstream was not covering all cases) this was working before, but now it is not. Now the configuration should be correct